tests/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Solution recommended by
+# http://python-guide-pt-br.readthedocs.io/en/latest/writing/structure/
+# import os
+# import sys
+# import airflow.utils.db
+#
+# sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../../")))
+
+# airflow.utils.db.initdb()
```

Comment:
What is this for? The link does not appear to work.

Reply:
Gah, the commented out code should have been deleted.